hack/cluster-up.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+source ./hack/common.sh
```

Comment:
Maybe just put `KUBEVIRT_DEPLOY_CDI=${KUBEVIRT_DEPLOY_CDI:-true}` directly here and not into common.

Reply:
Addressed, PTAL, thanks!